design/one-pager-fn-claim-conditions.md

```diff
@@ -0,0 +1,170 @@
+# Communication Between Composition Functions and the Claim
+
+* Owner: Dalton Hill (@dalton-hill-0)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Background
+
+### Desired Behavior
+Composition Function authors should be able to communicate with users. 
+Topics of communication include:
+- The status of underlying resources.
+- Errors that need to be resolved by the user.
+- Internal errors.
+
+### Existing Behavior
```

Comment:
A little detail about what things set status conditions and emit events would be useful here. I found myself going to look it up to refresh my memory. For the record:

* A Fatal result is emitted as a warning event associated with the XR.
* A Fatal result sets the `Synced: False` status condition. The result's message is included in the condition.
* Normal and warning results emit an event associated with the XR. The reason is `ComposeResources`.
* Normal and warning results don't influence any status condition.